Source/Libraries/NetCore/NetcoreCommands.cs

```diff
@@ -1,138 +1,129 @@
-        public const string REMOTE_EVENT_LOADGAMEDONE_NEWGAME = nameof(REMOTE_EVENT_LOADGAMEDONE_NEWGAME);
-        public const string REMOTE_EVENT_LOADGAMEDONE_SAMEGAME = nameof(REMOTE_EVENT_LOADGAMEDONE_SAMEGAME);
-        public const string REMOTE_EVENT_CLOSEEMULATOR = nameof(REMOTE_EVENT_CLOSEEMULATOR);
-        public const string REMOTE_EVENT_SHUTDOWN = nameof(REMOTE_EVENT_SHUTDOWN);
-        public const string REMOTE_OPENHEXEDITOR = nameof(REMOTE_OPENHEXEDITOR);
-
-        public const string GENERATEBLASTLAYER = nameof(GENERATEBLASTLAYER);
-        public const string SAVESAVESTATE = nameof(SAVESAVESTATE);
-        public const string LOADSAVESTATE = nameof(LOADSAVESTATE);
-        public const string REMOTE_LOADROM = nameof(REMOTE_LOADROM);
-        public const string REMOTE_CLOSEGAME = nameof(REMOTE_CLOSEGAME);
-        public const string REMOTE_PRECORRUPTACTION = nameof(REMOTE_PRECORRUPTACTION);
-        public const string REMOTE_POSTCORRUPTACTION = nameof(REMOTE_POSTCORRUPTACTION);
-
-        public const string REMOTE_BLASTTOOLS_GETAPPLIEDBACKUPLAYER = nameof(REMOTE_BLASTTOOLS_GETAPPLIEDBACKUPLAYER);
-
-        public const string ERROR_DISABLE_AUTOCORRUPT = nameof(ERROR_DISABLE_AUTOCORRUPT);
-
-        public const string REMOTE_KEY_SETSYNCSETTINGS = nameof(REMOTE_KEY_SETSYNCSETTINGS);
-        public const string REMOTE_KEY_SETSYSTEMCORE = nameof(REMOTE_KEY_SETSYSTEMCORE);
-
-        public const string EMU_OPEN_HEXEDITOR_ADDRESS = nameof(EMU_OPEN_HEXEDITOR_ADDRESS);
-        public const string EMU_GET_REALTIME_API = nameof(EMU_GET_REALTIME_API);
-        public const string EMU_GET_SCREENSHOT = nameof(EMU_GET_SCREENSHOT);
-        public const string REMOTE_EVENT_EMU_MAINFORM_CLOSE = nameof(REMOTE_EVENT_EMU_MAINFORM_CLOSE);
-        public const string REMOTE_EVENT_EMUSTARTED = nameof(REMOTE_EVENT_EMUSTARTED);
+        public const string CorruptCore = nameof(Basic) + "_" + nameof(CorruptCore);
+        public const string Vanguard = nameof(Basic) + "_" + nameof(Vanguard);
+        public const string Default = nameof(Basic) + "_" + nameof(Default);
+        public const string UI = nameof(Basic) + "_" + nameof(UI);
+        public const string KillswitchPulse = nameof(Basic) + "_" + nameof(KillswitchPulse);
+        public const string ResetGameProtectionIfRunning = nameof(Basic) + "_" + nameof(ResetGameProtectionIfRunning);
+        public const string ManualBlast = nameof(Basic) + "_" + nameof(ManualBlast);
+        public const string ApplyBlastLayer = nameof(Basic) + "_" + nameof(ApplyBlastLayer);
+        public const string ApplyCachedBlastLayer = nameof(Basic) + "_" + nameof(ApplyCachedBlastLayer);
+        public const string Blast = nameof(Basic) + "_" + nameof(Blast);
+        public const string StashKey = nameof(Basic) + "_" + nameof(StashKey);
+        public const string ErrorDiableAutoCorrupt = nameof(Basic) + "_" + nameof(ErrorDiableAutoCorrupt);
```

Comment:
This one sucks but can you fix this typo (diable)?

Reply:
Done here and Bizhawk (only other place it's used)

Reply:
Ah good I forgot it was only used in this way within Bizhawk so you didn't need to update everything,